resources/views/includes/notification.blade.php

```diff
@@ -13,7 +13,7 @@
                             : </span>{{ (session()->has('success')) ? session('success') : $errors->first() }}
                     </div>
                 @endif
-                @if(session()->has('no-internship'))
+                @if (Auth::user()->isStudent() && session()->has('no-internship'))
```

Comment:
Although this is functionally correct, maybe it is better to not even set the message in the session if the user is not a student. I think it happens in a middleware.

Reply:
Just removed this check (Auth::user()->isStudent()), I think it was unnecessary in the first place. I already check if the user is a student before setting the session in the middleware.

Reply:
Just removed this check (Auth :: user () -> isStudent ()), I think it was unnecessary in the first place. I already check if the user is a student before setting the session in the middleware.